packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -71,20 +128,45 @@ export class VaTelephone {
    * @return {string} - Combined phone number parts within the label seperated by
    * periods, e.g. "800-555-1212" becomes "8 0 0. 5 5 5. 1 2 1 2"
    */
-  static formatTelLabel(number: string): string {
-    return number
-      .split(/[^\d\w]+/)
-      .filter(n => n)
-      .map(chunk => (chunk === 'ext' ? 'extension' : chunk.split('').join(' ')))
-      .join('. ');
+  static formatTelLabel(number: string, vanity: boolean): string {
+    return vanity
+      ? number
+          .split(/[^\d]+/)
+          .filter(n => n)
+          .map(chunk => chunk.split('').join(' '))
+          .join('. ')
+      : number
+          .split(/[^\d\w]+/)
+          .filter(n => n)
+          .map(chunk =>
+            chunk === 'ext' ? 'extension' : chunk.split('').join(' '),
+          )
+          .join('. ');
```

Comment:
Could we shift the conditional logic into defining the regex that gets used instead?